config/default.yaml

```diff
@@ -131,6 +131,17 @@ plugins:
       # by default collapse builds or not
       collapse: true
 
+worker:
+    # https://github.com/taskrabbit/node-resque#multiworker-options
+    # minimum number of workers to spawn
+    minTaskProcessors: 1
+    # maximum number of workers to spawn
+    maxTaskProcessors: 10
+    # how often to check if the event loop is blocked
+    checkTimeout: 1000
+    # how long the event loop has to be delayed before considering it blocked
```

Comment:
might be good to put the units (ms)
but otherwise looks fine

Reply:
@tkyi good catch! I added the units to both timeout